stencila/hub
# Job UI

This PR resolves #454, updates #453 & resolves #434 

- Cancel job button opens a confirmation modal & sends posts a request to delete a job.
- Change sort order of to be newest job to oldest
- Make job row clickable to go to details
- Swap job token and status text information so that the job type is in the coloured token, and the status below it
- Hide Job runtime until the began field is populated. If Job still in progress calculate ongoing run time till present, if completed then calculate total run time.

**To Test**

Please check that a job:

> Hide Job runtime until the began field is populated. If Job still in progress calculate ongoing run time till present, if completed then calculate total run time.

**To Review**

- The cancel job & row clickable both require some javascript in order to work. This is using the existing view components (specifically `delete-confirm-modal`). Please check that this approach makes sense.
